document-generator-company-report/pom.xml

```diff
@@ -10,6 +10,16 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>document-generator-company-report</artifactId>
+    <properties>
+        <!-- Encoding -->
```

Comment:
these 2 comments are not needed as its self explanatory

Reply:
these are old comments , now removed.